### PR TITLE
Create theme and inaugurate pages

### PR DESCRIPTION
**Pull request by Saketh1499**

### CODE REVIEW

File: theme.html [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent"
+         aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+         <span class="navbar-toggler-icon" style="color:#f50000"><i class="fas fa-align-justify"></i></span>
+      </button>
+      <div class="collapse navbar-collapse" id="navbarSupportedContent">
+         <ul class="navbar-nav mr-auto  mx-auto navbar-right">
+            <li class="nav-item active">
+               <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item">
```
Comment (Sashi20) [inline theme.html:87]: Update the nav links to index.html wherever required

File: theme.html [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+               <a class="nav-link" href="#contact-us">Contact us</a>
+            </li>
+         </ul>
+      </div>
+   </nav> 
+   <br>
+ <h2><center><p style="color: purple" id="demo"></p></center></h2>
+
+<script>
```
Comment (Sashi20) [inline theme.html:129]: Add the script in custom.js file and please check if it is already present

File: theme.html [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+
+        <div class="mx-auto bd-highlight px-3">
+            
+        <div class="bs-example">
+            <ul class="nav nav-tabs">
+                <li class="nav-item">
+                    <a data-toggle="tab" id="home-tab" href="#home" class="nav-link active" role="tab" aria-controls="home" aria-selected="true">Theme of the Hackathon</a>
+                </li>
+                <li class="nav-item">
+                    <a data-toggle="tab" id="menu1-tab" href="#menu1" class="nav-link" role="tab" aria-controls="menu1" aria-selected="false">Submission Guidelines</a>
```
Comment (Sashi20) [inline theme.html]: Rename the tab ids relevant to the title of the tab

**Review by Sashi20 — CHANGES_REQUESTED** [review]

File: theme.html [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+       <p class="text-justify">Synfig Studio 2D Animation Hackathon. The theme for animation is:</p>
+        <center><img src="image/patch2.png"><br><br></center>
+        <!-- <div class="container">
+        <center>  <img src="image/patch-bg.png" alt="Snow" style="width:50%;">
+          <div class="bottom">The Panchatantra Stories</div>
+        </div> -->
+
+
+    <p><span >The Panchtantra is an ancient Sanskrit collection of stories. Team should create a 2D animation for these stories, using Synfig Studio software.</span></p>
+    <p><span style="font-weight: 400;">Please find the list of Panchatantra stories along the moral </span><a href="https://docs.google.com/spreadsheets/d/1-xssRyZwSbva25zyLBlV2iWCDj4wx8PnCuEzFS-Uqiw/edit#gid=0" target="_blank"><span style="font-weight: 400;">here</span></a><span style="font-weight: 400;">.</span></p>
```
Comment (Sashi20) [inline theme.html]: All the links should open in a new tab

File: theme.html [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+    
+  // If the count down is over, write some text 
+  if (distance < 0) {
+    clearInterval(x);
+    document.getElementById("demo").innerHTML = "EXPIRED";
+  }
+}, 1000);
+</script>
+
+  <section id="results">
```
Comment (Sashi20) [inline theme.html:160]: Remove bg-light class applied to the div

File: theme.html [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+    <p>You can have the stories interpreted according to the current situations happening across the world. For example the climate change, elections, pandemic situation, air pollution, etc. Please make sure that the moral of the story is not changed.</p>
+    </div>
+
+
+
+    <div id="menu1" class="tab-pane fade" role="tabpanel" aria-labelledby="menu1-tab">
+     <p>The participants have to submit their animation at <a href="https://courses.fossee.in">https://courses.fossee.in</a>. After confirming your participation in the Hackathon, your login credentials for submission will be generated and sent to you on the registered email ID.</p>
+        
+      <p><mark><span style="width:500px;height:100px;border:1px solid #000;">Please note: An email is sent to your registered email ID, where you have to confirm your participation in this Hackathon. Please make sure that this form is filled only by the team lead.</span></mark></p>
```
Comment (Sashi20) [inline theme.html]: Remove the border to this and any internal css applied

**Review by Sashi20 — CHANGES_REQUESTED** [review]

File: theme.html [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+            </li>
+         </ul>
+      </div>
+   </nav> 
+   <br>
+ <h2><center><p style="color: purple" id="demo"></p></center></h2>
+
+<script>
+// Set the date we're counting down to
+var countDownDate = new Date("Nov 25, 2020 00:00:00").getTime();
```
Comment (Sashi20) [inline theme.html]: This should be changed to the last date of the hackathon

**Review by Sashi20 — CHANGES_REQUESTED** [review]

File: theme.html (hunk 1 above)
Comment (Saketh1499) [reply theme.html:87]: Done

File: theme.html (hunk 2 above)
Comment (Saketh1499) [reply theme.html:129]: Done

File: theme.html (hunk 3 above)
Comment (Saketh1499) [reply theme.html]: Done

File: theme.html (hunk 4 above)
Comment (Saketh1499) [reply theme.html]: Done

File: theme.html (hunk 5 above)
Comment (Saketh1499) [reply theme.html:160]: Done

File: theme.html (hunk 6 above)
Comment (Saketh1499) [reply theme.html]: Done

File: theme.html (hunk 7 above)
Comment (Saketh1499) [reply theme.html]: Done

**Pull request merged**